src/Raven.Server/NotificationCenter/Indexing.cs

```diff
@@ -121,11 +133,19 @@ internal void UpdateIndexing(object state)
                     ((IndexingReferenceLoadWarning)referenceLoadsHint.Details).Update(tuple.indexName, tuple.warningDetails);
                 }
 
+                AlertRaised mismatchedReferencesAlert = null;
+                
+                if (_mismatchedReferencesLoadWarning != null) 
+                    mismatchedReferencesAlert = GetMismatchedReferencesAlert();
+                
                 if (indexOutputPerDocumentHint != null)
                     _notificationCenter.Add(indexOutputPerDocumentHint);
 
                 if (referenceLoadsHint != null)
                     _notificationCenter.Add(referenceLoadsHint);
+                
+                if (mismatchedReferencesAlert != null)
+                    _notificationCenter.Add(mismatchedReferencesAlert);
```

Comment:
Why not adding it immediately `if (_mismatchedReferencesLoadWarning != null)`?

Reply:
Done